src/main/java/ai/elimu/util/db/DbContentImportHelper.java

```diff
@@ -29,12 +29,8 @@
 import ai.elimu.model.content.StoryBookParagraph;
 import ai.elimu.model.content.Word;
 import ai.elimu.model.content.multimedia.Image;
-import ai.elimu.model.contributor.Contributor;
-import ai.elimu.model.contributor.LetterContributionEvent;
-import ai.elimu.model.contributor.LetterSoundCorrespondenceContributionEvent;
-import ai.elimu.model.contributor.NumberContributionEvent;
-import ai.elimu.model.contributor.StoryBookContributionEvent;
-import ai.elimu.model.contributor.WordContributionEvent;
+import ai.elimu.model.contributor.*;
```

Comment:
Wildcard imports are a bad practice

Reply:
This has not been addressed

Reply:
Even if I add it separately, it is taking as *

Reply:
Please review your IDE configuration, it probably collapsed them together by default

Reply:
resolved it